setup.py

```diff
@@ -21,8 +22,53 @@
                       runtime_library_dirs=['/usr/local/lib']))
     sys.argv.remove('--with-clime')
 
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+
 setuptools.setup(
-    setup_requires=['pbr'],
-    pbr=True,
+    name='keylime',
+    description=(
+        'TPM-based key bootstrapping and system '
+        'integrity measurement system for cloud'),
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    author='Keylime Community',
+    author_email='keylime@groups.io',
+    url='https://keylime.dev',
+    python_requires='>=3.6',
+    packages=['keylime'],
+    classifiers=[
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Information Technology',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: BSD License',
```

Comment:
This looks good to me @ashcrow , we recently changed to using Apache 2.0, could you just swap this out and we can land this PR..

Reply:
Switched to `License :: OSI Approved :: Apache Software License` per https://pypi.org/pypi?%3Aaction=list_classifiers